playground/main.js

```diff
@@ -55,11 +55,11 @@ const renderPlaygroundWidget = (options = {}) => {
       //    an array of tokens or a single token, depending on the
       //    initial configuration.
       else if (Array.isArray(res)) {
-        console.log(res);
+        console.log('===== playground widget success =====', res);
```

Comment:
I don't think we need this extra notice for success, as it'll complicate the assertion in our testcafe helpers.

I suggest keeping this as it was before, and letting the `JSON.parse(log[i])` logic handle parsing it so we can check the content through `checkConsoleMessages`

Reply:
+1

Reply:
Changed to parsing JSON